pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -471,7 +471,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 					},
 					{
 						Alert: "KubeVirtComponentExceedsRequestedMemory",
-						Expr:  intstr.FromString(fmt.Sprintf(`((kube_pod_container_resource_requests{namespace="%s",container=~"virt-controller|virt-api|virt-handler|virt-operator",resource="memory"}) - on(pod) group_left(node) container_memory_working_set_bytes{container="",namespace="%s"}) < 0`, ns, ns)),
+						Expr:  intstr.FromString(fmt.Sprintf(`kube_pod_container_resource_requests{namespace="%s",container=~"virt-controller|virt-api|virt-handler|virt-operator",resource="memory"} - on(node, namespace, pod, container) container_memory_working_set_bytes{image!=""} < 0`, ns)),
```

Comment:
Can you please elaborate on the reason you've replaced the matching filter from `container=""` to `image!=""`? The reason for using `container=""` was described [here](https://github.com/kubevirt/kubevirt/pull/7372#discussion_r835927944)
Is something changed since then?

Reply:
I changed the expression altogether because it was not working right now.
But from what I see, using the `container` or `image` filter has the same effect. I can revert that specific change, but what do you think about having a link for [this](https://github.com/kubevirt/kubevirt/pull/7372#discussion_r835927944) as a comment before the expression?

Reply:
@machadovilaca Sounds good to me. So let's keep the original functionality and lets add the comment.

Reply:
@enp0s3 added